tests/io/test_io.py

```diff
@@ -948,6 +948,7 @@ def test_frictionless_schema_parses_correctly(frictionless_schema):
         {"check": "not_nullable", "failure_case": "NaN"},
         {"check": "isin({1.0, 2.0, 3.0})", "failure_case": 1.1},
         {"check": "isin({1.0, 2.0, 3.0})", "failure_case": 3.8},
+        {"check": "dtype('float64')", "failure_case": "object"},
```

Comment:
what's this change for?

Reply:
Because we are not running coerce in schema_components anymore, we run `validate_column` in [here](https://github.com/unionai-oss/pandera/blob/dev/pandera/schema_components.py#L215). 

This means wrong values will be passed to `validate_column`. This error gets caught [here](https://github.com/unionai-oss/pandera/blob/dev/pandera/schemas.py#L2006-L2014) because the dtype is wrong.

It just adds another error to the error lists, which is why we have that additional error.